core/test-loader.ts

```diff
@@ -1,6 +1,7 @@
 import { ITestFixture, ITest, ITestCase } from "./_interfaces";
 import { FileRequirer } from "./_core";
 import { TestFixture, METADATA_KEYS } from "./alsatian-core";
+const getFunctionName = require("fn-name");
```

Comment:
I think that we probably don't need a dependency here, node 0.12 supports function.name property and as for legacy browser support, I'm not sure we need that :)